src/services/AuthService.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+const API_URL = "http://localhost:9040/";
+
+class AuthService {
+    async login(username, password, rememberMe) {
+
+        const response = await axios
+            .post(API_URL + "login", {
+                username,
+                password
+            });
+            
+        if(response.data.token) {            
+            localStorage.setItem("user", JSON.stringify(response.data));
+        }
+
+        if(rememberMe){
+            localStorage.setItem("username", username);
+            localStorage.setItem("password", password);
```

Comment:
I don't think we should store the password on the browser for security purposes. 
Take a look at [this](https://stackoverflow.com/questions/65299623/how-to-implement-remember-me-functionality-for-authentication-in-reactjs-when-i)